truss/tests/templates/server/test_model_wrapper.py

```diff
@@ -68,3 +68,20 @@ def test_model_wrapper_load_error_more_than_allowed(app_path, helpers):
         # Allow load thread to execute
         time.sleep(1)
         assert model_wrapper.load_failed()
+
+
+@pytest.mark.integration
+async def test_model_wrapper_streaming_timeout(app_path):
```

Comment:
it just feels like we're testing whether we're passing the config to the model_wrapper correctly. Could you add a test to these tests: https://github.com/basetenlabs/truss/blob/main/truss/tests/test_model_inference.py.

I'm imagining a test where we:
* set the streaming_read_timeout to something low (like a second)
* in the truss have it stream, but wait 2 seconds between yielding comment
* Ensure that a timeout error is logged